ani-cli

```diff
@@ -337,10 +341,12 @@ quality="${ANI_CLI_QUALITY:-best}"
 case "$(uname -a)" in
     *Darwin*) player_function="$(where_iina)" ;;                      # mac OS
     *ndroid*) player_function="${ANI_CLI_PLAYER:-android_mpv}" ;;     # Android OS (termux)
-    *neptune*) player_function="${ANI_CLI_PLAYER:-flatpak_mpv}" ;;    # steamdeck OS
     *MINGW* | *WSL2*) player_function="${ANI_CLI_PLAYER:-mpv.exe}" ;; # Windows OS
     *ish*) player_function="${ANI_CLI_PLAYER:-iSH}" ;;                # iOS (iSH)
-    *) player_function="${ANI_CLI_PLAYER:-mpv}" ;;                    # Linux OS
+    *) 
+      player_function="${ANI_CLI_PLAYER:-$(where_mpv)}"               # Linux OS
+      [ $? -eq 0 ] || exit $?
```

Comment:
This part is actually not necessary, since `dep_ch` will already exit

Reply:
The whole reason I did that so we can detect if either packaging format has mpv. I'm open to suggestions for better options.